app/src/main/java/com/theuhooi/uhooipicbook/modules/monsterlist/viewmodel/MonsterListViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.theuhooi.uhooipicbook.modules.monsterlist.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.theuhooi.uhooipicbook.modules.monsterlist.MonstersRepository
+import com.theuhooi.uhooipicbook.modules.monsterlist.entities.MonsterItem
+import com.theuhooi.uhooipicbook.repository.monsters.firebase.MonstersFirestoreClient
+
+class MonsterListViewModel : ViewModel() {
+
+    private val repository: MonstersRepository = MonstersFirestoreClient()
+
+    private val _monsterList = MutableLiveData<List<MonsterItem>>()
+    val monsterList: LiveData<List<MonsterItem>> = _monsterList
```

Comment:
あまり変わりませんが、Google I/O 2019のアプリでは、フィールドが増えない↓の書き方をしていたので、合わせたいです。
（SwiftでComputed Propertyを使えるときは使うのと同じ考えだと思います）

Ref:

- https://github.com/google/iosched/blob/master/mobile/src/main/java/com/google/samples/apps/iosched/ui/MainActivityViewModel.kt#L43-L45
- https://twitter.com/zetta1985/status/1263691483418202112?s=20

```suggestion
    private val _monsterList = MutableLiveData<List<MonsterItem>>()
    val monsterList: LiveData<List<MonsterItem>>
        get() = _monsterList
```

Reply:
なるほど、Twitterの方も拝見しましたがgetterの方が良さそうですね！